operator/src/main/java/oracle/kubernetes/operator/DomainRecheck.java

```diff
@@ -10,15 +10,13 @@
 import java.util.Objects;
 import java.util.Optional;
 import java.util.Set;
-import java.util.concurrent.atomic.AtomicBoolean;
```

Comment:
You've accidentally backed out a real change. Looks like you can just revert this file.